org.uqbar.project.wollok.tests/src/org/uqbar/project/wollok/tests/xpect/ConstructorDelegation.wlk.xt

```diff
@@ -92,14 +92,14 @@ class Superclase {
 class InvalidInstanceAccessOnConstructorDelegation inherits Superclase {
 	var x = 1
 	
-	// XPECT errors --> "Cannot access instance methods within constructor delegation." at "self"
+	// XPECT warnings --> "Cannot access instance methods within constructor delegation." at "self"
 	constructor() = super(self.getX()) 
 	
 	method getX() { return x }	
 }
 
 class InvalidInstanceAccessOnConstructorDelegation2 inherits Superclase {
-	// XPECT errors --> "Cannot access instance methods within constructor delegation." at "self"
+	// XPECT warnings --> "Cannot access instance methods within constructor delegation." at "self"
```

Comment:
No encuentro la relación entre el mensaje y el código.

Reply:
El mensaje ahora sería

`// XPECT warnings --> "Accessing instance methods within constructor delegation is dangerous because object might not be completely initialized." at "self"`

Tampoco me parece que tenga mucho sentido... pero es raro lo que intenta hacer como para entender si le puedo buscar un patrón para mejorar ese mensaje.